src/test/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowMonitorSpec.scala

```diff
@@ -83,26 +81,9 @@ class WorkflowMonitorSpec(_system: ActorSystem) extends TestKit(_system) with Fl
     }
   }
 
-  it should "fail a workflow if the method config can't be found" in withDefaultTestDatabase { dataSource =>
```

Comment:
I removed it because the behavior has changed to the point where the test does not make sense anymore.